locales/en.js

```diff
@@ -1,6 +1,94 @@
+   repliedTo : "replied to",
+   followed : "followed",
+   subscribedTo : "subscribed to",
+   and : "and",
+   others : "others",
+   pasteCode : "paste invite code here",
+   seeLocalUsers : "By default, Patchwork will only see other users that are on the same local area network as you.",
+   needPub : "In order to share with users on the internet, you need to be invited to a pub server.",
+   error : "Error",
+   ok : "Ok",
+   errorRedeemInvite : "An error occured while attempting to redeem invite.",
+   redeemInvite : "Redeem Invite",
+   Cancel : "Cancel",
+   confirm : "Confirm",
+   writeMessage : "Write a message",
+   publishing : "Publishing...",
+   Publish : "Publish",   
+   Compose : "Compose",
+   clickUnsubscribe : "Click to unsubscribe",
+   subscribe : "Subscribe",
+   subscribed : "Subscribed",
+   writeMessageChannel : "Write a message in this channel\n\n\n\nPeople who follow you or subscribe to this channel will also see this message in their main feed.\n\nTo create a new channel, type the channel name (preceded by a #) into the search box above. e.g #cat-pics",
+   writePrivate : "Write a private reply",
+   writePublic : "Write a public reply",
+   cannotLoadThead : "Cannot load thead",
+   thread : "Thread",
+   viewFullThread : "View full thread",
+   writePrivateMessage : "Write a private message \n\n\n\nThis can only be read by yourself and people you have @mentioned.",
+   selfAssigned : "Self Assigned",
+   assignedBy : "Assigned By",
+   editProfile : "Edit Your Profile",
+   clickUnfollow : "Click to unfollow",
+   friends : "Friends",
+   following : "Following",
+   followBack : "Follow Back",
+   follow : "Follow",
+   followers : "Followers",
+   likeToCall : "What whould you like to call",
+   writePublicMessage : "Write a public message",
+   joinPub: "+ Join Pub",
+   activeChannels : "Active Channels",
+   unsubscribe : "Unsubscribe",
+   local : "Local", //pubs
+   connectedPubs : "Connected Pubs",
+   whoToFollow : "Who to follow",
+   searchResults : "Search Results:",
+   searched : "Searched:",
+   noMatches : "No matches",
+   yourProfile: "Your profile",
+   aProfileLink : "a ProfileLink",
+   chooseProfileImage : "🖼 Choose Profile Image...",
+   chooseName : "Choose a name",
+   describeYourself : "Describe yourself (if you want)",
+   errorPublishing : "An error occured while attempting to publish about message.",
+   inReplyTo : "in reply to",
+   on : "on",
+   referencedFrom : "Referenced from",
+   newMessage : "New Message",
+   likedBy : "Liked by\n",
+   selfIdentifiesAs : "self identifies as ",
+   identified : "identified",
+   identifies : "identifies",
+   as : "as",
+   selfAssignedImage : "self assigned a display image",
+   assignedImageTo : "assigned a display image to",
+   selfAssignedDescription : "self assigned a description",
+   assignedDescriptionTo : "assigned a description to",
+   subscribedTo : "subscribed to",
+   unsubscribedFrom : "unsubscribed from"
+};
```

Comment:
From a developer perspective it would be nice if these key/values were "sorted". Just my 2 cents :)